res/values-fi/strings.xml

```diff
@@ -1066,9 +1066,9 @@
     <string name="PUSH_SPLIT">PUSH SPLIT</string>
     <string name="Level_Colors">Level Colors</string>
     <string name="click_info">Autoclick / Ultraclick do not work in FFA Classic!</string>
-    <string name="Hooked">Hooked</string>
+    <string name="Hooked">Koukussa</string>
     <string name="Be_pulled_by_a_hookshot_spell_for_10_seconds">Be pulled by a hookshot spell for 10 seconds</string>
-    <string name="Pack">Pack</string>
+    <string name="Pakata">Pack</string>
```

Comment:
Please return the original string name here. Change only the text in right side

Reply:
Done Jei, Thank u